lib/internal/process/warning.js

```diff
@@ -28,11 +29,16 @@ function setupProcessWarnings() {
 
   // process.emitWarning(error)
   // process.emitWarning(str[, name][, ctor])
```

Comment:
@ChALkeR could you please explain more? because this line is not that I changed in this pull-request, thanks :-)